starknet_py/net/client_models.py

```diff
@@ -829,13 +852,42 @@ def parsed_abi(self) -> Union[AbiDictListV2, AbiDictListV1]:
 
 
 @dataclass
-class SierraCompiledContract(SierraContractClass):
+class SierraCompiledContract(_SierraContract):
     """
     Dataclass representing SierraContractClass with required abi.
     """
 
     abi: str = field(default_factory=str)
 
+    @property
+    def parsed_abi(self) -> Union[AbiDictListV2, AbiDictListV1]:
+        load_abi: List = json.loads(self.abi)
+
+        if _is_abi_v2(load_abi):
+            return [
+                cast(
+                    AbiDictEntryV2,
+                    ContractAbiEntrySchemaV2(unknown=EXCLUDE).load(entry),
+                )
+                for entry in load_abi
+            ]
+
+        return [
+            cast(AbiDictEntryV1, ContractAbiEntrySchemaV1(unknown=EXCLUDE).load(entry))
+            for entry in load_abi
+        ]
+
+    def convert_to_sierra_contract_class(self) -> SierraContractClass:
+        """
+        Converts an instance of DeprecatedCompiledContract to DeprecatedContractClass.
+        """
```

Comment:
Is this docstring valid? I mean the returned type is `SierraContractClass` and in docstring is  `DeprecatedContractClass`.

Reply:
Fixed